example/multi-task/example_multi_task.py

```diff
@@ -76,7 +79,13 @@ class Multi_Accuracy(mx.metric.EvalMetric):
     """Calculate accuracies of multi label"""
 
     def __init__(self, num=None):
-        super(Multi_Accuracy, self).__init__('multi-accuracy', num)
+        self.num = num
```

Comment:
Better swap line 82 and 83.

Reply:
Parent constructor calls reset, which relies on self.num which doesn't exist if I put it as the first line.